geometry_msgs/PoseStamped pose
float64[] feature
int32 initialization_id # -1 if not initialization
int32 observation_id
int32 timestep
bool negative_observation
